src/System.Collections/ref/System.Collections.cs

```diff
@@ -96,7 +96,8 @@ public partial class Dictionary<TKey, TValue> : System.Collections.Generic.IColl
         public virtual void OnDeserialization(object sender) { }
         public bool Remove(TKey key) { throw null; }
         public bool Remove(TKey key, out TValue value) { throw null; }
-        public void TrimExcess(int capacity) { }
+		public int RemoveAll(System.Predicate<System.Collections.Generic.KeyValuePair<TKey, TValue>> match) { throw null; }
+		public void TrimExcess(int capacity) { }
```

Comment:
Indentation in this two lines seems to be off.

Reply:
Fixed